go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -84,6 +87,30 @@ var (
 	}{}
 
 	parseAndValidateCreate = func(cmd *cobra.Command, args []string) error {
+		if createOptions.ParamsFile != "" {
+			paramsFile, err := os.ReadFile(createOptions.ParamsFile)
+			if err != nil {
+				return err
+			}
+			createParams := &vtctldatapb.LookupVindexCreateParams{}
+			err = json2.UnmarshalPB(paramsFile, createParams)
+			if err != nil {
+				return err
+			}
+			return parseCreateParams(createParams)
```

Comment:
It seems like the we're making it so everything is in the file or nothing. That's OK, but we should enforce that with mutually exclusive pflag options.